lib/install/is-fs-access-available.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var fs = require('fs')
+var process = require('process')
```

Comment:
Yup. I didn't even know I could require process until I went to write these tests =D